### PR TITLE
🔀🌐📝nl_NL Module translations and docs

### PR DESCRIPTION
**Pull request by Piet2001**
Dutch module translations
Dutch module docs
en_US POI

### CODE REVIEW

File: src/modules/extendedCallWindow/i18n/nl_NL.js
Comment (Suchty112) [inline src/modules/extendedCallWindow/i18n/nl_NL.js:103]: @KBOE2 Sollen da nicht eigentlich erstmal nur die standard rein die es gibt?
 Shouldn't there really only be the standard pure ones available?
default

File: src/modules/missionHelper/settings.ts
Comment (Suchty112) [inline src/modules/missionHelper/settings.ts:127]: Why First Responder only nl_NL? The chance exists in every country and can be "triggered" by any car except ambulances or HEMS?

Wieso First Responder nur nl_NL? Die Chance gibt es in jedem land und kann von jedem auto außer Ambulancen oder HEMS "ausgelöst" werden?

Comment (JRH-1997) [reply src/modules/missionHelper/settings.ts:127]: No, it's !==, because it's not in NL.

Comment (Suchty112) [reply src/modules/missionHelper/settings.ts:127]: Ups. Mein Fehler. Es ist schon spät ^^ . Stimmt ihr habt das ja nicht. seh ich auch grad
Oops. My bad. It's getting late ^^ . That's not true. I can see that now.

File: src/modules/extendedCallWindow/i18n/nl_NL.js
Comment (JRH-1997) [reply src/modules/extendedCallWindow/i18n/nl_NL.js:103]: The "Politie Noodhulp" and "Politie Overig" is a wish of many users. It's also already on planning.

**Review by KBOE2 — CHANGES_REQUESTED** [review]
As i got write access on Piet's fork, I will run the linter on the source branch directly once and merge afterwards :wink:

**Comment by Piet2001** [comment]
I have send you an invite

**Comment by KBOE2** [comment]
don't ask me why, but I already had write access before ^^

However, wil lbe merged here as soon as Travis built successfully

**Comment by Piet2001** [comment]
maybe because it is an fork of your repo

**Review by KBOE2 — APPROVED** [review]

**Pull request merged**
